nomt/src/beatree/ops/update/leaf_updater.rs

```diff
@@ -162,21 +160,29 @@ impl LeafUpdater {
         // note: if we need a merge, it'd be more efficient to attempt to combine it with the last
         // leaf of the bulk split first rather than pushing the ops onwards. probably irrelevant
         // in practice; bulk splits are rare.
-        let last_ops_start = self.build_bulk_splitter_leaves(new_leaves);
+        if self.gauge.body_size() > LEAF_BULK_SPLIT_THRESHOLD {
+            self.try_build_leaves(new_leaves, LEAF_BULK_SPLIT_TARGET)
+        }
+
+        // If the gauge is over LEAF_NODE_BODY_SIZE at least one node
+        // respecting the halful requirement will always be created.
+        //
+        // The only scenarion where the first leaf node creted will have a
+        // body_size below the target is when there is an item who makes the
```

Comment:
"when there is an item which causes the size to jump from underfull to overfull".

Reply:
Grammatically, I understand the correction, but the phrase does not state my what I intended.

The point is to jump from  **below the target** to overfull, not from **underfull** to overfull.